src/main/java/ch/wisv/areafiftylan/extras/consumption/model/ConsumptionMap.java

```diff
@@ -37,7 +38,7 @@
 
     @NonNull
     @ManyToMany(fetch = FetchType.EAGER)
-    private Collection<Consumption> consumptionsMade;
+    private List<Consumption> consumptionsMade;
```

Comment:
Fair point. I reversed the consumptionsMade to Collection, as changing just the Roles to a Set seems to do the trick. I checked for all occurences, no problems found.